gulpfile.js/index.js

```diff
@@ -62,7 +61,6 @@ exports.imagemin = function( cb ) {
  * Gulp task to minify and combine svg's.
  */
 exports.svg = gulp.series(
-	svgstore,
 	svgmin,
```

Comment:
Is this task still relevant? I would think the `gulp svg` command should probably be unnecessary now as svgs should be entirely handled by Webpack. I mentioned svgstore before because it was clearly related to the sprite creation but again, svgs should be fully built/managed by Webpack going forward.

Reply:
@aaemnnosttv Now that I'm looking more at this (and considering a previous comment where I mention minifying), should we be adding feature parity to Webpack that Gulp used to have? We previously had an SVG minifier setup within our build step but the original issue makes no mention of re-creating the minifying logic in Webpack, only that the previous setup should be removed.

Reply:
Related gulp tasks were removed but I'm keeping this convo unresolved to discuss Webpack feature parity with the previous Gulp setup.

Reply:
Good question! It looks like SVGO is included with `@svgr/webpack`, we may just need to configure it
```
├─┬ @svgr/webpack@5.4.0
│ └─┬ @svgr/plugin-svgo@5.4.0
│   └── svgo@1.3.2
```
https://react-svgr.com/docs/configuration-files/

I don't think we need to go to any extreme lengths to preserve 100% parity with what we had before, but establishing the configuration file with any necessary/sensible values would be great 👍

Reply:
Looks like we were using the default SVGO config with `removeViewbox` set to `false`, this is pretty easy to mimic with a config file. This has been added to `.svgo.yml`